src/main/java/frc/robot/subsystems/pivot/PivotIOSim.java

```diff
@@ -10,49 +10,48 @@
 import edu.wpi.first.math.system.plant.DCMotor;
 import edu.wpi.first.wpilibj.simulation.DCMotorSim;
 import edu.wpi.first.wpilibj.simulation.SingleJointedArmSim;
+import frc.robot.Constants.PivotConstants;
 
 
 public class PivotIOSim implements PivotIO {
-    private final double pivotLength = 0-9;
-    private final double pivotMass = 0-9;
-    private SingleJointedArmSim leaderPivotMotorSim = new SingleJointedArmSim(DCMotor.getKrakenX60(1), 0, 0, 0, 0, 0, false, 0);
-    private SingleJointedArmSim followerPivotMotorSim = new SingleJointedArmSim(DCMotor.getKrakenX60(1), 0, 0, 0, 0, 0, false, 0);
+    private final double pivotLength = PivotConstants.PIVOT_LENGTH;
+    private final double pivotMass = PivotConstants.PIVOT_MASS;
+    private final double pivotGearing = PivotConstants.PIVOT_GEARING;
+
+    private SingleJointedArmSim pivotSim = new SingleJointedArmSim(DCMotor.getKrakenX60(2), pivotGearing, pivotMass, pivotLength, 0, 0, true, 0);
 
```

Comment:
u probably also will want to create a ProfiledPIDController to control the position of the pivotSim. I can help u with this at the meeting